packages/devtools_app/test/memory_screen_test.dart

```diff
@@ -368,12 +370,40 @@ void main() {
       await tester.pumpAndSettle(_twoSeconds);
     }
 
+    Future<void> checkGolden(String goldenFilename, {Key key}) async {
+      // Await delay for golden comparison.
+      await tester.pumpAndSettle(const Duration(seconds: 2));
```

Comment:
nit: be consistent about using pumpAndSettle and pumpAndSettleTwoSeconds.

Reply:
Oops, was messing with duration and forgot reverting. Done.